<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sistema</class>
 <widget class="QMainWindow" name="Sistema">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>500</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Arboria-Black</family>
    <pointsize>24</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(85, 170, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="botao_cadastrar_aviao">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>390</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arboria-Black</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 170, 0);</string>
    </property>
    <property name="text">
     <string>Cadastrar Avião</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_numeroRegistro">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>120</y>
      <width>187</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Número de Registro</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_numeroRegistro_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>180</y>
      <width>187</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Modelo</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_numeroRegistro_3">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>240</y>
      <width>224</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Quantidade de assentos</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_numeroRegistro_4">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>300</y>
      <width>224</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Quantidade de assentos</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_numeroRegistro_5">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>330</y>
      <width>187</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>especiais</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>120</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>180</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>240</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>300</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_titulo_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>551</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Sistema de passagens aéreas</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botao_voltar">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>0</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arboria-Black</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 0, 255);
color: rgb(255,255,255);</string>
    </property>
    <property name="text">
     <string>Voltar</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
